app/components/forms/user-payment-info-form.js

```diff
@@ -0,0 +1,106 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { validPhoneNumber } from 'open-event-frontend/utils/validators';
+import { pick } from 'lodash-es';
+
+export default Component.extend(FormMixin, {
```

Comment:
Lets start moving to ES6 style classes for all new resources. 
https://github.com/fossasia/open-event-frontend/pull/3169/files